Forget password module

THIWANKA07 opened the pull request:
ushan merge ekk dpn mchn

THIWANKA07 committed:
Forget password module

Ushan-Dilusha commented:
please resolve conflicts @IT21261350

Ushan-Dilusha commented on src/component/pages/forgotPasswordPage/index.ts:
```diff
@@ -0,0 +1,3 @@
+import forgotPasswordPage from "./forgotPasswordPage";
+
+export { forgotPasswordPage };
```
```suggestion
export { forgotPasswordPage };

```

Ushan-Dilusha commented on src/component/pages/resetPasswordPage/index.ts:
```diff
@@ -0,0 +1,3 @@
+import resetPasswordPage from "./resetPasswordPage";
+
+export { resetPasswordPage   };
```
```suggestion
export { resetPasswordPage   };

```

Ushan-Dilusha committed:
Update src/component/pages/forgotPasswordPage/index.ts

Ushan-Dilusha committed:
Update src/component/pages/resetPasswordPage/index.ts

Ushan-Dilusha committed:
Merge branch 'develop' into object_identifier_header_and_instructions

Ushan-Dilusha committed:
merge fixes

Ushan-Dilusha committed:
Merge branch 'develop' into object_identifier_header_and_instructions

OvindaNamal committed:
Modification
1. login page
2. drawing page
3. registration page
4. foget password
5. home page
6. number idintofy page

Ushan-Dilusha committed:
Merge branch 'develop' into object_identifier_header_and_instructions

OvindaNamal committed:
Add back button to teacher Registration page
